src/plugins/kibana/public/visualize/editor/agg_group.js

```diff
@@ -40,6 +42,29 @@ uiModules
           if (count < schema.max) return true;
         });
       });
+
+      const drake = dragula({
+        containers: $('.vis-editor-agg-group', $el).toArray(),
+        moves(el, source, handle) {
```

Comment:
Well. Affordances. I'd be fine with that if and only if we could clearly convey that aggregations are draggable. Right now the button combined with the cursor when its hovered over makes that pretty clear.